packages/api/database/migrations/1605981580411_refactor_disclaimers_status_schema.js

```diff
@@ -0,0 +1,25 @@
+/** @type {import('@adonisjs/lucid/src/Schema')} */
+const Schema = use('Schema');
+const { disclaimersTypes } = require('../../app/Utils');
+
+class RefactorDisclaimersStatusSchema extends Schema {
+	up() {
+		this.alter('disclaimers', (table) => {
+			// alter table
+			table
+				.enu('type', Object.values(disclaimersTypes))
+				.defaultTo(disclaimersTypes.PRIVACYPOLICY)
+				.notNullable()
+				.alter();
+		});
+	}
+
+	down() {
+		this.alter('disclaimers', (table) => {
+			// reverse alternations
+			table.dropColumn('type');
```

Comment:
O rollback está funcionando normalmente? Testa dar um `adonis migration:rollback` pra testar.

Pelo o que entendi, no `up` alteramos um campo existente, logo, no `down`, este campo **deveria ser alterado para exatamente como era antes.**

Outra coisa a se observar é o nome da migration. Creio que ficaria mais significativo se tivéssemos algo como `change type to enum in disclaimers table`

Reply:
Testei o comando `adonis migration:rollback` e funcionou.
Nessa migration não estou mudando o tipo do campo, estou só
colocando o array de opções no padrão das outras migrations, com o `Object.values(disclaimersTypes)`

Reply:
Veja que o commit com a mudança do método down gerou um warning nos testes do CI.